lesson19/src/css/styles.css

```diff
@@ -0,0 +1,369 @@
+  background-size: cover;
+  width: 100%;
+  height: 100%;
+}
+
+.mainvisual-textarea{
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.mainvisual-textarea__catch{
+  font-size: 4rem;
+  letter-spacing: 0.1em;
+  text-align: center;
+}
+
+.mainvisual-textarea__sub{
+  background: -webkit-linear-gradient(top, #8360c3, #2ebf91);
+  background: linear-gradient(to top, #8360c3, #2ebf91);
+  background-clip: text;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.mainvisual-paginationbullet{
+  display: flex;
+  margin: 20px 0 0 -10px;
+}
+
+.paginationbullet__bar{
+  width: calc(100% / 5);
+  height: 6px;
+  background-color: #fff;
+  display: block;
+  margin-left: 10px;
+  border-radius: 3px;
+  box-shadow: -5px -5px 10px 0px rgba(255, 255, 255, 0.5), 5px 5px 10px 0px rgba(0, 0, 0, 0.3);
+  position: relative;
+  cursor: pointer;
+}
+
+.paginationbullet__bar[aria-current="true"]::before{
+  content: '';
+  width: 100%;
+  height: 100%;
+  display: block;
+  border-radius: 3px;
+  background-color: #6DAD97;
+  animation: bar 3s linear;
+}
+
+.slider-pagination{
+  margin-top: 10rem;
+  font-size: 2rem;
+}
+
+.separation{
+  margin: 0 2rem;
+}
+
+.slider-button{
+  position: absolute;
+  display: flex;
+  justify-content: space-between;
+  z-index: 1;
+  width: 80%;
+}
+
+.arrow{
+  display: inline-block;
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  background: transparent;
+  text-indent: -9999px;
+  border-top : 3px solid #fff;
+  border-left: 3px solid #fff;
+  transition: all .3s ease-in-out;
+  text-decoration: none;
+  color: transparent;
+  cursor: pointer;
+}
+
+.arrow::before{
+  display: block;
+  height: 200%;
+  width: 200%;
+  margin: -50% 0 0 -50%;
+  content: "";
+  transform: rotate(45deg);
+}
+
+.arrow.previous{
+  transform: rotate(-45deg);
+  left: 0;
+}
+
+.arrow.next{
+  transform: rotate(135deg);
+  right: 0;
+}
+
+.arrow:disabled{
+  border-color: rgba(255, 255, 255, 0.5);
+  pointer-events: none;
+}
+
+.arrow:hover{
+  border-color: #1B4025;
+}
+
+@keyframes slide-fade{
+  0% {
+    transform: translateX(0%);
+    opacity: 1;
+  }
+  40% {
+    opacity: 0.7;
+  }
+  100% {
+    transform: translateX(-100%);
+    opacity: 0;
+  }
+}
+
+@keyframes animationZoom {
+  0% {
+    opacity: 0;
+  }
+  20% {
+    opacity: 1;
+  }
+  100% {
+    transform: scale(1.1)
+  }
+}
+
+@keyframes bar {
+    0% {
+        width: 0;
+        transform: translateX(0)
+    }
+
+    50% {
+        width: 50%;
+    }
+
+    80% {
+        width: 80%;
+    }
+
+    100% {
+        width: 100%;
+    }
+}
+
+.news {
+	width: 80%;
+	max-width: 960px;
+	margin: 80px auto auto auto;
+}
+
+.tab-list {
+	display: flex;
+	flex-direction: row;
+	cursor: pointer;
+	align-items: flex-end;
+  margin-top: 36px;
+}
+
+.tab_list_item {
+	width: calc(100% / 5);
+}
+
+.tab-button {
+	width: 100%;
+	padding: 0.8rem 2rem;
+	border-radius: 0.6rem 0.6rem 0 0;
+	border: #fff;
+}
```

Comment:
nits
How about a pointer on the tab buttons as well?

Reply:
Thank you! Certainly it is difficult to know if you can click without a pointer.
This has been fixed.

[492f821](https://github.com/nyattsu72/hanson-frontend-cource/pull/27/commits/492f8213d13cfddbbeceafc338bbefe22e7a65da)